packages/core/helpers/coveUpdateWorker.ts

```diff
@@ -14,10 +16,25 @@ export const coveUpdateWorker = config => {
   }
   let genConfig = config
 
-  genConfig = update_4_24_3(genConfig)
-  genConfig = update_4_24_4(genConfig)
-  genConfig = update_4_24_5(genConfig)
-  genConfig = update_4_24_7(genConfig)
+  if (!genConfig.version) {
+    genConfig.version = '0.0.1'
+  }
+
+  if (versionNeedsUpdate(genConfig.version, '4.24.3')) {
```

Comment:
can you create a helper function for this? like you could create an array of touples. like [versionNumber, versionFunction]

const versions = [['4.24.3', update_4_24_3]...] and loop through that to do the versioning? would be much easier to update.